static/scss/answers/cards/product-prominentimage-clickable.scss

```diff
@@ -40,6 +40,9 @@
   {
     width: 100%;
     padding: calc(var(--yxt-base-spacing) * 3/4) var(--yxt-base-spacing);
+    display: flex;
+    flex-direction: column;
+    height: 100%;
```

Comment:
why do we need height: 100%? can we use flex-grow?

Reply:
ah, you're right, I'll try that instead